src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -344,6 +344,30 @@ public function setAnnotatedClassCache(array $annotatedClasses)
     }
 
     /**
+     * Return a list of classes to be cached away when the container is build.
+     *
+     * If a class contains annotations, add it to getAnnotatedClassesToCompile()instead.
+     *
+     * @return string[]
+     */
+    public function getClassesToCompile()
```

Comment:
Is it still worth it when using php7 ? If not, i don't think we should still encourage using it.